src/styles/reset.scss

```diff
@@ -0,0 +1,133 @@
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
```

Comment:
Remove all useless tags which you don`t use in your work: table/caption and so on

Reply:
I'm not sure it's necessary. I think it's okay to leave it like this.

Reply:
@mpohorenyi Actually I agree here with @YaroslavYarynych, it's a good practice to have in your codebase only that code that actually does something. It can be fine in some test works, but in real project, it's super important. Imagine a team with 5 devs and each of you leaves a few lines of useless code in the project every week. What will happen with the code in the year? And the good projects live more than a year.